app/src/main/res/layout/fragment_top_up.xml

```diff
@@ -41,7 +41,7 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_marginStart="32dp"
-            android:text="--"
+            android:text="---"
```

Comment:
maybe it can have a minWidth so that the skeleton doesn't change size when loading